src/lib/TranscriptEditor/MediaPlayer/PlayerControls.module.css

```diff
@@ -1,8 +1,12 @@
 @value color-darkest-grey, color-light-grey, color-labs-red from '../colours.module.css';
 
 .playerControls {
```

Comment:
This section needs rethinking. The player controls and the grouped buttons within could probably be centred and responsive to narrow screens with flexbox and + wrap? Not sure how fiddly that'll be.

I find here the Timebox is too far away from the player + progress bar. Also the huge grid of buttons is overwhelming. An alternative for mobile _could be_ player controls + timebox stuck to the bottom of the screen?

Probably needs more UX input rather than winging it. We're undoing a lot of stuff from the first round of UX in this PR so let's not get carried away 😛 



<img width="321" alt="screen shot 2019-02-21 at 14 36 16" src="https://user-images.githubusercontent.com/9170575/53176666-76990200-35e6-11e9-8eb6-ac08d3bea865.png">


Also, any idea why the MediaPlayer size doesn't update properly a lot of the time?

Reply:
Yeah, we can rethink it, I reckon we can just do a first pass in this PR and then optimise later. Not good to try and do everything at once.